modules/hcp-ocp-differences.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/index.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="hcp-ocp-differences_{context}"]
+= Differences between {hcp} and {product-title}
+
+{hcp-capital} is a form factor of {product-title}. Hosted clusters and the standalone {product-title} clusters are configured and managed differently. See the following tables to understand the differences between {product-title} and {hcp}:
+
+
+[id="cluster-creation_{context}"]
+== Cluster creation and lifecycle
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|You install a standalone {product-title} cluster by using the `openshift-install` binary or Assisted Installer.
+|You install a hosted cluster by using the `hypershift.openshift.io` API resources such as `HostedCluster` and `NodePool`, on an existing {product-title} cluster.
+
+|===
+
+[id="cluster-configuration_{context}"]
+== Cluster configuration
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|You configure cluster-scoped resources such as authentication, API server, and proxy by using the `config.openshift.io` API group.
+|You configure resources that impact the control plane in the `HostedCluster` resource.
+
+|===
+
+[id="etcd-encryption_{context}"]
+== etcd encryption
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|You configure etcd encryption by using the `APIServer` resource with AES-GCM or AES-CBC. For more information, see "Enabling etcd encryption".
```

Comment:
This is where CTRL+F "Enabling etcd encryption" becomes tricky if the the text does not match the metadata of a link.